### PR TITLE
Updates to sample-stack CI scripts & docs

### PR DESCRIPTION
**Pull request by groeges**
Added a missing CI script (ci_wait.sh) that casued the travis build to fail.
Updated CI script to specify the latest version of the Appsody CLI to use.
Added some initial docs on how to use the sample stack repo and CI scripts.

### CODE REVIEW

File: ci/env.sh [hunk 1]
```diff
@@ -141,15 +141,10 @@ logged() {
...
-#this is the default list of repos that we need to build index for
-if [ -z "$REPO_LIST" ]; then
-    export REPO_LIST="experimental incubator stable"
-fi
-
 # image registry org for publishing stack
 if [ -z "$IMAGE_REGISTRY_ORG" ]
 then
-    export IMAGE_REGISTRY_ORG=appsody
+    export IMAGE_REGISTRY_ORG=myorg
```
Comment (kilnerm) [inline ci/env.sh]: Should we maybe fail rather than defaulting in this case?

**Review by kilnerm — CHANGES_REQUESTED** [review]
Looks good barring one question.

File: ci/env.sh (hunk 1 above)
Comment (groeges) [reply ci/env.sh]: Updated to fail.

**Review by kilnerm — APPROVED** [review]
LGTM

**Pull request merged**